File: 2-semestre/uri-online-judge/c/1145.c
#include <stdio.h>
 
int main() {
    int x, y, i, cont = 1;

    scanf("%d %d", &x, &y);

    for (i = 1; i <= y; i++) {
        if (x != cont) {
            printf("%d ", i);
            cont++;
        } else {
            printf("%d\n", i);
            cont = 1;
        }
    }

    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1145.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const linha = lines.shift().split(" ");
const x = parseInt(linha[0]);
const y = parseInt(linha[1]);

let cont = 1;
let result = "";

for (let i = 1; i <= y; i++) {
    if (cont !== x) {
        result += i + " ";
        cont++;
    } else {
        result += i + "\n";
        cont = 1;
    }
}

result = result.replace(/^\s+|\s+$/g, '');

console.log(result);
